test/ServerTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ZendTest\Expressive\Swoole;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Zend\Expressive\Swoole\Server;
+use Swoole\Http\Server as SwooleHttpServer;
+
+class ServerTest extends TestCase
+{
+
+    public function setUp()
+    {
+        $this->container = $this->prophesize(ContainerInterface::class)->reveal();
+    }
+
+    public function testCreateSwooleServer()
```

Comment:
Note to self: this is "testCreateSwooleServerCreatesAndReturnsSwooleHttpServerInstance".

Reply:
Note to self: we need another test indicating that subsequent calls to the method return the same instance.